src/frontend/shortlist/src/App.vue

```diff
@@ -10,6 +10,23 @@ const apiClient = new ShortlistApi("https://api.shortlist.nyc");
 const router = useRouter();
 const appSessionStore = sessionStore();
 
+function sendASchoolCardToTrash(payload) {
```

Comment:
To match with the other pr - can this be:
```suggestion
function markSchoolAsRejected(payload) {
```

Reply:
Updated in the new branch